static/src/stylesheets/module/facia-garnett/item-types/_fc-item--type-media.scss

```diff
@@ -119,6 +119,11 @@
         color: $special-report-alt-dark;
     }
 
+    &:hover,
+    .u-faux-block-link--hover {
+        background-color: darken($special-report-alt-faded, 2%);
```

Comment:
### Media card in container without a palette - hovering state
| Before      | After      |
|-------------|------------|
| ![image](https://user-images.githubusercontent.com/19683595/226298130-fb4ff63b-827c-4045-b668-0d3105d1e99e.png) | ![after][] |